tests/trinity/core/p2p-proto/test_peer_block_header_validator_api.py

```diff
@@ -94,11 +95,13 @@ def mk_header_chain(length):
 @pytest.mark.asyncio
 async def test_les_peer_get_headers_round_trip(les_peer_and_remote,
                                                params,
+                                               monkeypatch,
                                                headers):
     peer, remote = les_peer_and_remote
+    request_id = 9
+    monkeypatch.setattr(trinity.protocol.les.exchanges, 'gen_request_id', lambda: request_id)
```

Comment:
If you go back in to make changes, can you add a `TODO` comment here indicating that we *should* be able to retrieve the `request_id` by reading the request message from the peer.

Reply:
In the process of writing the `TODO` I figured out what I was doing wrong :man_facepalming: (The peer already awaits the commands, I just need to subscribe to the decoded messages). Fixed.